config/manifests/bases/metallb-operator.clusterserviceversion.yaml

```diff
@@ -54,6 +54,7 @@ spec:
   - email: rbryant@redhat.com
     name: Russell Bryant
   maturity: alpha
+  minKubeVersion: 1.26.0
```

Comment:
what fails without that?

Reply:
operator-sdk fires a warning (it's also in the commit description):
 Warning: Value : (metallb-operator.v0.0.0) csv.Spec.minKubeVersion is not informed. It is recommended you provide this information.